docs/reference/filebeat/filebeat-input-azure-blob-storage.md

```diff
@@ -199,18 +202,17 @@ This attribute contains the details about a specific container like `name`, `max
 
 This is a specific subfield of a container. It specifies the container name.
 
+## `batch_size` [attrib-batch_size-abs]
+
+This attribute specifies the `page size` for the response. In earlier versions, this value was derived from `max_workers`, but with the latest update, `batch_size` is now an independent setting. For backward compatibility, if `batch_size` is not explicitly defined, it will default to a value based on `max_workers`. This attribute can be configured at both the root and container levels. When defined at both levels, the container-level setting takes precedence.
```

Comment:
Is `page size` an attribute? Should `page size` actually be `batch_size`? Or should it be `page_size` like `batch_size`? Or just "page size" (not formatted as code)?

Reply:
I think this should be "page size" (not code formatted).